frontend/src/app/features/readers/pdf-reader/pdf-reader.component.html

```
@@ -405,14 +404,14 @@
   }
 </ng-template>
 
-<ng-template #btnBase let-id="id" let-icon="icon" let-title="title" let-active="active" let-pressed="pressed" let-className="className" let-isOverflow="isOverflow" let-size="size">
+<ng-template #btnBase let-id="id" let-icon="icon" let-title="title" let-active="active" let-pressed="pressed" let-isOverflow="isOverflow" let-size="size">
   @if (isOverflow) {
     <button class="overflow-item" [class.active]="active" (click)="performAction(id); closeToolbarOverflow()">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)